src/globals.css

```
@@ -1,4 +1,121 @@
 :root {
-    --main-white: #FFFFFF;
-    --main-blue: #0023b1;
+    --primary-blue: #0023b1;
+    --primary-green: #3bb100;
+    --primary-yellow: #b19400;
+    --primary-orange: #b15e00;
+    --primary-red: #b10f00;
+    --primary-grey: #afafaf;
+    --primary-white: #FFFFFF;
+
+    --secondary-white: #ededed;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    background-color: var(--secondary-white);
+}
+
+.size-small {
+    font-size: 12px;
+    width: 30%;
+    padding-top: 8px;
+    padding-bottom: 8px;
+}
+
+.size-medium {
+    font-size: 16px;
+    padding-top: 12px;
+    padding-bottom: 12px;
+    width: 55%;
+}
+
+.size-large {
+    font-size: 20px;
+    padding-top: 16px;
+    padding-bottom: 16px;
+    width: 100%;
+}
+
+.bg-blue {
+    background-color: var(--primary-blue);
+}
+
+.bg-green {
+    background-color: var(--primary-green);
+}
+
+.bg-yellow {
+    background-color: var(--primary-yellow);
+}
+
+.bg-orange {
+    background-color: var(--primary-orange);
+}
+
+.bg-red {
+    background-color: var(--primary-red);
+}
+
+.bg-grey {
+    background-color: var(--primary-grey);
+}
+
+.bg-white {
+    background-color: var(--primary-white);
+}
+
+.text-blue {
+    color: var(--primary-blue);
+}
+
+.text-green {
+    color: var(--primary-green);
+}
+
+.text-yellow {
+    color: var(--primary-yellow);
+}
+
+.text-orange {
+    color: var(--primary-orange);
+}
+
+.text-red {
+    color: var(--primary-red);
+}
+
+.text-grey {
+    color: var(--primary-grey);
+}
+
+.text-white {
+    color: var(--primary-white);
+}
+
+.border-blue {
+    border: 3px solid var(--primary-blue);
+}
+
+.border-green {
+    border: 3px solid var(--primary-green);
+}
+
+.border-yellow {
+    border-color: 3px solid var(--primary-yellow);
```

border et pas border-color
```suggestion
    border: 3px solid var(--primary-yellow);
```